environments/india/inventory.ini.j2

```diff
@@ -41,7 +41,7 @@ es_data
 {{ __pillow2__ }}
 {{ __kafka1__ }} kafka_broker_id=0
 {{ __pgbouncer0__ }}
-{{ __pgbouncer1__ }}
+{{ __pgbouncer1__ }} pgbouncer_processes=2
```

Comment:
Do we also specify the connection limit, or is that automatically calculated? (I didn't see where it's calculated if it is)

Reply:
I think this is getting calculated here: https://github.com/dimagi/commcare-cloud/pull/4454/files#diff-ab93faae53b13f08b16910656c2ca624924ae1562e0a159d9d325fb3d4fc83edR120.

We checked when we rolled it out and each pgbouncer process has the pool size set to 145 = 290 // 2